src/nih_wayfinding/app/scripts/profiles/profile-preference-options.js

```diff
@@ -39,13 +39,13 @@
 
             speedOpts: [{
                 text: 'About average (2.5 mph)',
-                value: 2.5
+                value: 1.1176
```

Comment:
Since you're using miles for the navigation progress stuff, I'll just move the conversion to the parameter-building logic.